src/Sulu/Bundle/TranslateBundle/Build/TranslationBuilder.php

```diff
@@ -32,14 +32,11 @@ public function getName()
 
     public function getDependencies()
     {
-        return ['database'];
+        return [];
     }
 
     public function build()
     {
-        foreach ($this->locales as $locale) {
-            $this->execCommand('Translations', 'sulu:translate:import', ['locale' => $locale]);
-            $this->execCommand('Translations', 'sulu:translate:export', ['locale' => $locale]);
-        }
+        $this->execCommand('Translations', 'sulu:translate:export');
```

Comment:
Didn't we say we introduce a Composer Script Handler for that? So that this file can also be removed?